sdk/feature-set/src/lib.rs

```diff
@@ -1095,6 +1099,7 @@ lazy_static! {
         (remove_accounts_executable_flag_checks::id(), "Remove checks of accounts is_executable flag SIMD-0162"),
         (lift_cpi_caller_restriction::id(), "Lift the restriction in CPI that the caller must have the callee as an instruction account #2202"),
         (disable_account_loader_special_case::id(), "Disable account loader special case #3513"),
+        (accounts_lt_hash::id(), "enables lattice-based accounts hash #3333"),
```

Comment:
Once we have a SIMD, we need to update this to SIMD number instead of PR number.